Exercises_7_a_b_c.cpp

```diff
@@ -0,0 +1,98 @@
+
+#include "stdafx.h"
+using namespace cv;
+using namespace std;
+
+
```

Comment:
in my opinion,exercise 7 does not need something like argv.
Can I omit this item?
just in exercise 7,of course.